cmd/kubeadm/app/cmd/token_test.go

```diff
@@ -226,13 +226,13 @@ func TestNewCmdToken(t *testing.T) {
 		},
 		{
 			name:          "valid: delete from --kubeconfig",
-			args:          []string{"delete", "abcdef.1234567890123456", "--dry-run", "--kubeconfig=" + fullPath},
+			args:          []string{"delete", "abcdef.1234567890123456", "--dry-run=client", "--kubeconfig=" + fullPath},
```

Comment:
hi, i don't think the cmd/kubeadm changes in this PR are valid.
did you happen to use an automated search and replace query to replace these?

please note that different components (kubelet, kubeadm, or even their sub-commands) might have different dry-run implementations.

/hold

Reply:
Yes, please remove this change since I don't think it applies to the right command.